uwblueprint/supportive-housing
# Log record many residents frontend

## Notion task link


[Log Record Can Have Many Residents](https://www.notion.so/uwblueprintexecs/Log-Record-Can-Have-Many-Residents-f54cc5e2220545c187223caf3a0a743e)



## Implementation description
* Frontend: allow user to multiselect residents when creating and editing logs; display multiple residents in record in log records table



## Steps to test
1. Migrate database: `bash ./scripts/flask-db-upgrade.sh`
2. Create log: (a) Try making a log with no residents selected - this should give an error. (b) Select multiple residents from the dropdown, play around with adding and removing residents from the selected list. After submitting the new log, you should see all of the residents you selected in the corresponding record in the log records table.
4. Edit log: When you edit a log, the previously selected residents should be pre-selected for you. Try removing residents and adding residents to the list. After submitting the edited log, you should see your changes in the corresponding record in the log records table.
5. Delete log



## What should reviewers focus on?
* 


## Checklist
